### ISSUE TITLE
Goblins and Ogres don't spawn in the 1.18.1 version

### ISSUE THREAD

**Issue by RollingNiko** [superlord9362/GoblinsAndDungeons#20]
i can't find a single goblins and dungeons mob, not even in the castle

**Comment by superlord9362** [comment]
Unable to recreate. Give me more info on the world and try to recreate the problem.

**Comment by Temosaurus** [comment]
I my be able to expand on this issue. I started up a brand new world and there was a goblin castle close to my spawn. Having never encountered one, I went to investigate it from a distance. As I got close though, the boss bar for the goblin king suddenly disappeared, which I thought was weird. After reaching the castle I found that there were no goblins in it at all, the castle was completely empty. To figure out the issue I went in creative and spawned a goblin king, which worked. After leaving and reentering the chunk however, the boss bar appeared for a second before vanishing, with the goblin king nowhere to be found.
To summarize it seems goblins will shortly despawn after entering a chunk with them. Whether this is a mod incompatibility or just a general bug is unknown.

**Comment by superlord9362** [comment]
Fixed for 1.18.2
